components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -201,10 +203,12 @@ tweaking their $case/namelist_scream.xml file.
       <rad_frequency hgrid="ne512np4">3</rad_frequency>
       <rad_frequency hgrid="ne1024np4">3</rad_frequency>
       <do_aerosol_rad>true</do_aerosol_rad>
+      <do_aerosol_rad COMPSET=".*SCREAM*noSPA*">false</do_aerosol_rad>
```

Comment:
Change to pattern match to

<do_aerosol_rad COMPSET=".*SCREAM.*noSPA*">false</do_aerosol_rad>